modules/administration_manual/pages/configuration/files/encryption/enabling-user-key-encryption.adoc

```diff
@@ -0,0 +1,132 @@
+* When having data shared with a group and group membership changes after the share is established, subsequently added users will not be able to open the shared data unless the owner will share it again.
+
+== Enable User-Key Based Encryption
+
+To enable User-Key based encryption requires two steps:
+
+. xref:enable-the-encryption-app[Enable the encryption app]
+. xref:enable-and-configure-user-key-based-encryption[Enable and configure User-Key based Encryption]
+
+[TIP] 
+====
+We strongly encourage you to put your server in single user mode before setting up encryption.
+
+To do so, run the following command:
+
+....
+sudo -u www-data php occ maintenance:singleuser --on
+....
+====
+
+[[enable-the-encryption-app]]
+=== Enable the Encryption App
+
+include::enable-encryption.adoc[leveloffset=+1,tag=enable-encryption-app-via-command-line]
+
+[[enable-and-configure-user-key-based-encryption]]
+=== Enable and Configure User-Key Based Encryption
+
+To enable and configure User-Key based encryption, you need to:
+
+. Enable the default encryption module app
+. Enable encryption
+. Enable the user-key, using the following command:
+. Encrypt all data
+. Turn off the single user mode
+
+The following example shows how to carry out these steps.
+
+[source,console]
+....
+sudo -u www-data php occ encryption:enable
+sudo -u www-data php occ encryption:select-encryption-type user-keys
+sudo -u www-data php occ encryption:encrypt-all
+sudo -u www-data php occ maintenance:singleuser --off
+....
+
+[[how-to-enable-users-file-recovery-keys]]
+== How To Enable User File Recovery Keys
+
+Once a user has encrypted their files, if they lose their ownCloud password, then they lose access to their encrypted files, as their files will be unrecoverable. 
+It is not possible, when user files are encrypted, to reset a user's password using the standard reset process.
+If so, you'll see a yellow banner warning:
+
+""
+Please provide an admin recovery password; otherwise, all user data will be lost.
+""
+
+To avoid all this, create a recovery key. 
+To do so, go to menu:Settings[Admin > encryption] and set a recovery key password.
+
+image:configuration/files/encryption10.png[image]
+
+You then need to ask your users to opt-in to the Recovery Key. 
+For the users to do this, they need to go to the ''**Personal**'' page and enable the recovery key. 
+This signals that they accept that the admin might have a way to decrypt their data for recovery reasons. 
+If they do _not_ do this, then the recovery key won't work for them.
+
+image:configuration/files/encryption7.png[image]
+
+For users who have enabled password recovery, give them a new password and recover access to their encrypted files, by supplying the Recovery Key on the Users page.
+
+image:configuration/files/encryption8.png[image]
+
+You may change your recovery key password.
```

Comment:
oh, this statement says, at least for me for the first time, that a recovery key is password protected...
If this is the case, it should be mentioned above/earlier as (security) statement.